src/components/terminal.tsx

```
@@ -103,6 +66,56 @@ export default function Terminal() {
     return () => mediaQuery.removeEventListener("change", handleChange);
   }, [theme]);
 
+  // Determine which theme classes to use
+  const currentTheme =
```

ESLint

[error] 'currentTheme' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)